_posts/2015-03-22-experiences-from-elasticon.markdown

```diff
@@ -0,0 +1,70 @@
+
+Conferences are a great way to recharge and get inspired by meeting people and hearing about the latest and greatest. That’s what the dev community is all about! Here’s my experience:
+
+# Arriving to San Francisco
+San Francisco is a beautiful city with endless opportunities to explore. I started taking pictures as soon as I arrived and posting them to the Elastic{ON} app - the app by the way was a fantastic idea. Everyone was interacting long before we ever made it to the kick off party. From the Embarcadero, to Alcatraz island, to the top of Coit Tower and over the Golden Gate Bridge I logged over 1000 photos and have countless amazing memories. 
+
+# Ok, The conference
+If I had to describe the conference with one word it would be inspiring. Inspiring is a powerful word, but here’s the proof: 1300 attendees from 35 countries gathered at one location, and for the very first time the entire Elastic team was united under one roof, creating a unique opportunity to explore insights to search, data analysis and aggregations.
+
+![Elasticon](/assets/econ-photos.png)
+
+# The Keynote
+The conference kicked off with a Keynote where the CEO, Steven Schuurman, and the creator, founder and CTO, Shay Banon, took us on a trip through the history of Elasticsearch after which, they made two big announcements: first Elasticsearch has been rebranded to [Elastic](https://www.elastic.co), and second [Found](https://www.found.no/), the Elasticsearch as a service company has been acquired by Elastic. Elasticsearch will remain as a core product for Elastic, but the new brand will encompass the broader scope of their suite of products and services.
+
+# Elastic{ON} had three main tracks
+
+#### Track 1 - ELK in the Wild(A track dedicated to companies sharing how they are using Elasticsearch, Logstash and Kibana)
+
+By far this was the most popular track due to presentations given by companies like Cisco, USGS, Mozilla, Verizon, Yale, Github, Wikimedia, Yelp, Fico, Netflix and Zendesk among others. It was literally standing room only or no room at all. A couple of presentations I was able to squeeze in:  
+
+1. **”Tectonic Shifts, Tweets, & Tactical Data Analysis”** [presented by Michelle Guy from USGS](https://www.elastic.co/elasticon/2015/sf/tectonic-shifts-tweets-and-tactical-data-analysis/?view=1)
+
+>> Apparently, instrument based earthquake monitoring systems may take up to 10 minutes to broadcast information regarding the epicenter of an earthquake. USGS has a project currently in Beta which analyzes tweet trends during seismic activity which has reduced the feedback loop to less than 2 minutes. The tweet data is geocoded and put into Elasticsearch, a pretty map in a Kibana dashboard does the rest. How cool is that?
+
+
+2. **”Scaling Elasticsearch for Production at Verizon: 500 Billion Documents & Counting”** [presented by Bhaskar Karambelkar from Verizon](https://www.elastic.co/elasticon/2015/sf/scaling-elasticsearch-for-production-at-verizon/?view=1)
```

Comment:
This doesn't actually seem to register as '2' on my screen. It thinks you are starting a new list.